packages/ra-data-simple-prisma/src/createHandler.ts

```diff
@@ -102,13 +102,13 @@ export const createHandler = async <Args extends CreateArgs>(
         //    });
         // (data) mediaIds: [1, 2, 3] => postToMediaRels: { create: [{connect: {media: {id: 1}}}, {connect: {media: {id: 2}}}, {connect: {media: {id: 3}}}] }
 
-        const foreignCreateKey = firstKey(foreignConnect); // => postToMediaRels
+        const foreignCreateKey = Object.keys(foreignConnect)[0]; // => postToMediaRels
```

Comment:
was causing type errors

Reply:
mmmh, I thought it was a different issue actually, firstKey is using `Object.keys(o)[0]` so that should be alright, can you show me the Ts ussue?
<img width="266" alt="Screenshot 2024-03-18 at 10 40 37" src="https://github.com/codeledge/ra-data-simple-prisma/assets/4820803/c82b56d0-96e4-4bd1-a2f8-3ba6ed4cf4c0">